nnpdf_data/nnpdf_data/theory.py

```diff
@@ -0,0 +1,51 @@
+"""
+This module provides an unique source and definition for all the possible parameters
```

Comment:
```suggestion
This module provides an unique source and definition for some the possible parameters
```
These are not all the one that can be specified right?

Reply:
Let's say for "all the parameters that define a theory".

The importance people want to give to each parameter is up to people. And it is up to the interface between nnpdf and eko (i.e., pineko) to decide how the parameters are translated. So the parameters from eko don't need to have a duplicate here. But if you would need them to define and reproduce a theory, it would be better if it is here as well.

If `eko` decides to change how the parameter is called, here it will still be called `n3lo_ad_variation` and it would be up to `pineko` how to translate it.